### DISCUSSION TITLE
Support for IntelliJ IDEA (Chat)

### DISCUSSION THREAD

**Discussion by eazypaulCode** [community/community#56548] [Copilot Conversations]
Yea would be great to support GitHub Copilot Chat in JetBrains IDEs!

**Comment by Akash1134** [comment]
Hi @eazypaulCode ,

Thank you for posting in the Copilot community! We value your feedback and our team is paying attention to the feedback, feature requests & bugs we are hearing from users in the Community Discussions.

At this time, the public preview for Copilot for Chat is for VS and VSCode users. However, we'll share your feedback with the concerned team so that can look into it.

**Reply by ChaseApptentive** [reply]
Do you all have a timeline for getting IntelliJ caught up with VSCode? I'm sure it must be on your roadmap.

**Comment by TylorWu** [comment]
I hope soon

**Comment by tzutoo** [comment]
Any update on Intellij IDEA Github Copilot Chat support?

**Comment by Faq** [comment]
https://blog.jetbrains.com/idea/2023/06/ai-assistant-in-jetbrains-ides/
